tuf/client/updater.py

```diff
@@ -1495,14 +1496,21 @@ def _get_metadata_file(self, metadata_role, remote_filename,
           metadata_spec_version = metadata_signable['signed']['spec_version']
           metadata_spec_major_version = int(metadata_spec_version.split('.')[0])
           code_spec_major_version = int(tuf.SPECIFICATION_VERSION.split('.')[0])
-
+          
           if metadata_spec_major_version != code_spec_major_version:
             raise tuf.exceptions.UnsupportedSpecificationError(
                 'Downloaded metadata that specifies an unsupported '
                 'spec_version.  This code supports major version number: ' +
                 repr(code_spec_major_version) + '; however, the obtained '
                 'metadata lists version number: ' + str(metadata_spec_version))
 
+          #report to user if minor versions do not match, continue with update
+          if pkg_resources.parse_version(metadata_spec_version) != pkg_resources.parse_version(tuf.SPECIFICATION_VERSION):
```

Comment:
If you compare the full versions you don't need to parse them. But if I understand correctly, you don't want to compare the full version, just the minor version.  The major version you already compare above and patch/fix version you don't care about.

Also, your log message says that one version (part) is higher than the other, so you will need to compare with `>` operator. 

I suggest you parse both `metadata_spec_version` and `tuf.SPECIFICATION_VERSION` only once and then use the major parts for the first (existing) check and the minor parts for the second (your new) check.